app/src/main/kotlin/com/softteco/template/ui/feature/forgotPassword/ForgotPasswordViewModel.kt

```diff
@@ -57,57 +68,81 @@ class ForgotPasswordViewModel @Inject constructor(
         State()
     )
 
-    private fun handleSuccess() {
-        snackBarState.value = SnackBarState(
-            textId = R.string.check_email,
-            show = true,
-        )
+    init {
```

Comment:
It seems it could be simplified if it moved to the top 
```
    private val ctaButtonState = emailState.map { email ->
        email.text.validateInputValue(FieldType.EMAIL) is FieldState.Valid
    } 
```